.scripts/auto_update.py

```
@@ -0,0 +1,77 @@
+        if build_touched and not config_touched:
+            print(
+                f"🛠️  Code changed but version not bumped. Bumping patch for {addon}..."
+            )
+            try:
+                bump_version(
+                    addon, "patch", "Maintenance: Automated dependency/base update"
+                )
+                # We need to output this for the workflow to know what to release
+                with open(os.environ.get("GITHUB_OUTPUT", "output.txt"), "a") as f:
+                    f.write(f"updated_addon={addon}\n")
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)